terraform/ec2/default/amazon-cloudwatch-agent.json

```
@@ -0,0 +1,20 @@
+{
+  "agent": {
+    "debug": true,
+    "region": "$REGION"
+  },
+  "traces": {
+    "traces_collected": {
+      "app_signals": {
```

This config has been changed, update it everywhere where applicable (only within the scope of this PR)
Also check if anything else has been changed, check public docs
```suggestion
      "application_signals": {
```

---

Done

---

Also removed `enabled`